### ISSUE TITLE
Restore a single database

### ISSUE THREAD

**Issue by vitobotta** [databacker/mysql-backup#121]
Hi! From looking at the script it seems like it cannot restore a single database even if I back up as one file per each schema? What is the easiest/best way to restore a single database then? I am looking to use this in Kubernetes. Thanks!

**Comment by deitch** [comment]
For now, that is correct. It isn't all that hard to add it though. If you feel like a PR, that would be great! :-)

**Comment by vitobotta** [comment]
Thanks for confirming. I'll see if I have a chance and am able to do it :)

**Comment by vitobotta** [comment]
BTW how do I specify DB_NAMES as env variable? space delimited list or? I am configuring this with a cronjob in K8s

**Comment by deitch** [comment]
Correct. Let's keep it consistent with the var used for backup, which is `DB_NAMES`
